### PR TITLE
Added More info in README.md

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Added More Info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # test_repository
 To test.
+To test the lambda function's operation.
```
Comment (dilipchauhan14) [inline README.md:3]: Added More Info

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # test_repository
```
Comment (dilipchauhan14) [inline README.md:1]: Usual_1

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
...
 To test.
```
Comment (dilipchauhan14) [inline README.md:2]: Usual_2

**Review by dilipchauhan14 — COMMENTED** [review]
Marked Usuals.

**Review by sammarth-kapse — APPROVED** [review]

**Pull request merged**